spec/spec_helper.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+require 'paperclip/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.